.eslintignore

```diff
@@ -1,4 +1,5 @@
 coverage/*
 dist/
 node_modules/
-src/segment.jsx
\ No newline at end of file
+src/segment.jsx/
+utils.jsx
```

Comment:
For some reason, course/data/utils.jsx and only this file says that it is not included under the tsconfig file. 
![image](https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/322346/7da32726-3441-478a-929c-50420b605e30)

Reply:
Renaming utils.tsx fixed the issue.  Props to @davidjoy for suggesting this might be the issue!